pages/ee/introduction.adoc

```diff
@@ -0,0 +1,48 @@
+[[gravitee-enterprise-api-platform]]
+= Enterprise Edition
+:page-sidebar: ee_sidebar
+:page-permalink: ee/ee_overview.html
+:page-folder: ee
+:page-toc: false
+:page-description: Gravitee Enterprise Edition support, installation, licensing, and versioning - overview
+:page-keywords: Gravitee, API Platform, Enterprise Edition, documentation, manual, guide, reference, api
+
+The Enterprise Edition (EE) of Gravitee.io offers additional features on top of the fantastic open source Community Edition of the platform, bringing many new benefits and exciting features to help you enable effortless control over every aspect of your organization's API ecosystem.
+
+== EE components
+
+The Enterprise Edition offers the following additional modules and plugins:
+
+* Gravitee link:https://docs.gravitee.io/am/current/am_overview_introduction.html[Alert Engine^] (AE) is an EE module you can use to monitor, analyze, and prevent incidents such as SLA breaches or spikes in traffic happening in the future. To activate AE in APIM and/or AM, you need to apply an link:{{ '/ee/ee_license.html' | relative_url }}[EE license] to your APIM/AM installation.
+* Other EE components include some additional policies and plugins shipped with APIM and AM, as follows:
++
+--
+EE plugins for APIM that require an EE license:
+
+* link:https://docs.gravitee.io/apim/3.x/apim_policies_data_logging_masking.html[Data Logging Masking Policy^]
+* link:https://docs.gravitee.io/apim/3.x/apim_policies_assign_metrics.html[Assign Metrics Policy^]
+
+EE plugins and features for AM that require an EE license:
+
+* link:https://docs.gravitee.io/am/current/am_userguide_mfa_factors_http.html[HTTP Flow Identity Provider]
+* link:https://docs.gravitee.io/am/current/am_userguide_enterprise_identity_provider_cas.html[CAS Identity Provider^] (now shipped separately like all other EE plugins)
+* link:https://docs.gravitee.io/am/current/am_userguide_enterprise_identity_provider_kerberos.html[Kerberos Identity Provider^] (now shipped separately like all other EE plugins)
+* link:https://docs.gravitee.io/am/current/am_userguide_enterprise_identity_provider_saml2.html[SAML 2.0 Identity Provider^]
+* link:https://docs.gravitee.io/am/current/am_userguide_mfa_factors_fido2.html[MFA with FIDO2^]
+* link:https://docs.gravitee.io/am/current/am_userguide_mfa_risk_based.html[Risk-based MFA^]
+* link:https://docs.gravitee.io/am/current/am_userguide_alerts_overview.html[Alerts^] (de facto the option to integrate the link:https://docs.gravitee.io/am/current/am_overview_introduction.html[Alert Engine^] EE module in AM)
```

Comment:
Shouldn't we use asciidoc syntax to create these links ?
Two methods can be used (prefer the first one if possible):
```
link:{% link pages/apim/3.x/installation-guide/installation-guide.adoc %} [here]
link:{{ '/apim/3.x/apim_installguide.html' | relative_url }}[here]
```

Reply:
Thanks @phiz71 - I guess here you mean Jekyll’s [{% link %} tag](https://jekyllrb.com/docs/liquid/tags/#link) :) Not AsciiDoc syntax. I am all for us using the first format as it is the best for sustaining the correct permalinks and respecting the base URL. However, I think it's not worth the time/effort to fix these now, being that we're moving away from Jekyll imminently, and we have a lot of inconsistent usage of links throughout the current docs.

Reply:
👌🏻 I'm ok with that. You can merge :)